### PR TITLE
Supminn lint fixes

### LINKED ISSUE #813: ✨ [Feature request]: Code cleanup `Common` folder
### Is your feature request related to a problem? Please describe.

Fix the eslint and formatting issue for common folder file formatting.

### Describe the solution you'd like.

Fix the eslint and formatting issue for common folder file formatting.

### Describe alternatives you've considered.

.

### Screenshot / Screenshare

_No response_

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by supminn**
> First thing, PLEASE READ THIS: [ReactPlay Code Review Checklist](https://github.com/reactplay/react-play/wiki/ReactPlay-Code-Review-Checklist)

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

- Fixes #813 

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] eslint fixes for files within `common` folder

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

# Checklist:

- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works

## Note

I am getting the following error when I try to run `yarn start`. Thus I am unable to test my changes.
image

### CODE REVIEW

**Comment by supminn** [comment]
All the linting errors within the `common` folder have been fixed. However, `yarn start` is giving an error.

File: src/common/footer/ExtendedFooter.jsx [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
 import { MdManageSearch } from 'react-icons/md';
 import { FaDiscord } from 'react-icons/fa';
 import { BsTwitter, BsGithub } from 'react-icons/bs';
 import LogoLight from 'images/img-logo-light.svg';
-import { SearchContext } from 'common/search/search-context';
+import { useSearchContext } from 'common/search/search-context';
 
 const ExtendedFooter = () => {
-  const { showShareModal, setShowShareModal } = useContext(SearchContext);
+  const { showShareModal, setShowShareModal } = useSearchContext();
```
Comment (atapas) [inline src/common/footer/ExtendedFooter.jsx:10]: Why don't we need to pass search context here?

Comment (supminn) [reply src/common/footer/ExtendedFooter.jsx:10]: I have replaced `useContext` with a custom hook `useSearchContext`
It creates an abstraction, but in turn uses `SearchContext`

Comment (atapas) [reply src/common/footer/ExtendedFooter.jsx:10]: I checked that now!

A couple of questions: 

- Can you conform all the errors in common are resolved?
- How have you tested such that the functionalities are not broken?

Comment (supminn) [reply src/common/footer/ExtendedFooter.jsx:10]: Hi @atapas 
I tested out the app and it seems to be working fine with all my changes.

However, we do get the pop-up to fix the other lint issues (from `play`) folder. This PR contains lint fixes for files within the `common` folder.

**Comment by atapas** [comment]
Ok Thanks @supminn. Merging this to code clean up.

**Pull request merged**